modules/tinymce/src/plugins/table/main/ts/ui/TableDialog.ts

```diff
@@ -113,7 +122,11 @@ const onSubmitTableForm = (editor: Editor, tableElm: HTMLTableElement | null | u
     }
 
     if (Obj.size(modifiedData) > 0) {
-      applyDataToElement(editor, tableElm, data);
+      applyDataToElement(editor, tableElm, data, {
+        border: Obj.has(modifiedData, 'border'),
+        bordercolor: Obj.has(modifiedData, 'bordercolor'),
+        cellpadding: Obj.has(modifiedData, 'cellpadding')
+      });
```

Comment:
Could just be me but I think I would it would cleaner/easier to read if the applicable cell properties object was a separate variable and passed in e.g. `applyDataToElement(editor, tableElm, data, cellProperties);`